internal/cmd/release/main.go

```diff
@@ -582,38 +584,22 @@ func fetchRegistryImageAndImageID(plugin *plugin.Plugin) (string, string, error)
 	if err != nil {
 		return "", "", err
 	}
-	imageName := fmt.Sprintf("ghcr.io/%s/plugins-%s-%s", release.GithubOwnerBufbuild, identity.Owner(), identity.Plugin())
-	cmd, err := dockerCmd("manifest", "inspect", "--verbose", imageName+":"+plugin.PluginVersion)
+	imageName := fmt.Sprintf("ghcr.io/%s/plugins-%s-%s:%s", release.GithubOwnerBufbuild, identity.Owner(), identity.Plugin(), plugin.PluginVersion)
+	parsedName, err := name.ParseReference(imageName)
 	if err != nil {
 		return "", "", err
 	}
-	var bb bytes.Buffer
-	cmd.Stdout = &bb
-	if err := cmd.Run(); err != nil {
-		// this may occur if this runs prior to publishing a newly added plugin
-		return "", "", nil //nolint:nilerr
-	}
-	type manifestJSON struct {
-		Descriptor struct {
-			Digest string `json:"digest"`
-		} `json:"Descriptor"` //nolint:tagliatelle
-		SchemaV2Manifest struct {
-			Config struct {
-				Digest string `json:"digest"`
-			} `json:"config"`
-		} `json:"SchemaV2Manifest"` //nolint:tagliatelle
-	}
-	var result manifestJSON
-	if err := json.Unmarshal(bb.Bytes(), &result); err != nil {
-		return "", "", fmt.Errorf("unable to parse docker manifest inspect output: %w", err)
-	}
-	descriptorDigest := result.Descriptor.Digest
-	if descriptorDigest == "" {
-		return "", "", errors.New("unable to parse descriptor digest from docker manifest inspect output")
-	}
-	imageDigest := result.SchemaV2Manifest.Config.Digest
-	if imageDigest == "" {
-		return "", "", errors.New("unable to parse image config digest from docker manifest inspect output")
-	}
-	return fmt.Sprintf("%s@%s", imageName, descriptorDigest), imageDigest, nil
+	remoteImage, err := remote.Image(parsedName, remote.WithAuthFromKeychain(githubkeychain.Keychain))
```

Comment:
This configures auth for ghcr.io container images with the GITHUB_TOKEN env var (we're already using this to authenticate with github).